Cargo.toml

```
@@ -8,6 +8,8 @@ members = [
     "extensions/scarb-snforge-test-collector",
     "utils/create-output-dir",
     "utils/scarb-build-metadata",
+    "utils/scarb-macro-attributes",
+    "utils/scarb-macro-interface",
```

for example:

```suggestion
    "plugins/scarb-macro-attributes",
    "plugins/scarb-macro-interface",
```

to avoid the `utils` group to grow infinitely beyond private-ish utility crates?